xc7/yosys/synth.tcl

```diff
@@ -7,16 +7,16 @@ yosys -import
 
 # Map (actually rename) explicitly instantiated IBUFs/OBUFs in order to
 # distinguish them from the inferred ones.
-techmap -map $::env(symbiflow-arch-defs_SOURCE_DIR)/xc7/techmap/iob_map.v
+#techmap -map $::env(symbiflow-arch-defs_SOURCE_DIR)/xc7/techmap/iob_map.v
```

Comment:
Why are these commented out? Should there be an issue to add them back?

Reply:
That is actually a very subtle question.  The bottom line is: ROI based designs need these lines, and ROI-less design **cannot** have these lines.  

The reason things worked before on the basys3 is that all of the IO attributes could be left as defaults, and on the nexys video, that is not the case.  More tcl logic is required make this synth script work for both ROI and ROI-less designs, and I want to get a clocking design working before I go back and fix it.

I also think I want to split off the changes to the `synth.tcl` into a smaller PR anyways.  I'm working on splitting off the other changes now, like the `PULLTYPE` fix, so that reviews can focus on the changes.

Reply:
Fix is going in for iob inference here: https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1339